exercises/word-search/canonical-data.json

```diff
@@ -0,0 +1,594 @@
+{
+  "exercise": "word-search",
+  "version": "1.0.0",
+  "comments": [
+    "Grid rows and columns are 1-indexed.",
+    "An expected value of -1 indicates that some sort of failure should occur."
+  ],
+  "cases": [
+    {
+      "description": "Should locate words written left to right",
+      "property": "search",
+      "grid": [
+        "jefblpepre",
+        "camdcimgtc",
+        "oivokprjsm",
+        "pbwasqroua",
+        "rixilelhrs",
+        "wolcqlirpc",
+        "screeaumgr",
+        "alxhpburyi",
+        "jalaycalmp",
+        "clojurermt"
+      ],
+      "wordsToSearchFor": [
+        "clojure"
+      ],
+      "expected": [
+        {
+          "start": {
+            "column": 1,
+            "row": 10
+          },
+          "end": {
+            "column": 7,
+            "row": 10
+          }
+        }
+      ]
+    },
+    {
+      "description": "Should locate words written right to left",
+      "property": "search",
+      "grid": [
+        "jefblpepre",
+        "camdcimgtc",
+        "oivokprjsm",
+        "pbwasqroua",
+        "rixilelhrs",
+        "wolcqlirpc",
+        "screeaumgr",
+        "alxhpburyi",
+        "jalaycalmp",
+        "clojurermt"
+      ],
+      "wordsToSearchFor": [
+        "clojure",
+        "elixir"
+      ],
+      "expected": [
+        {
```

Comment:
It occurs to me that the only indication that the first element in this array indicates the position of `"clojure"` and the second that of `"elixir"` is convention.

I'm sorry that I cannot speak for all tracks, but I know that in Go the tests at least arrange such that output is a map whose keys are strings and corresponding values are what we have here (start/end column/row).

So I would like to ask about that. It seems unclear to leave it up to convention.

Another interesting item is that it appears the Go track is the only track that asks for multiple words at once. It's true that that matches the description. If it has to be that way (rather than simply changing the description), I would like to make sure that interface is solid before proceeding